packages/grid/src/GridMetricCalculator.ts

```diff
@@ -99,14 +99,12 @@ class GridMetricCalculator {
     let x = 0;
     for (let i = 0; i < startCount && i < totalCount; i += 1) {
       coordinates.set(i, x);
-      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      x += sizeMap.get(i)!;
+      x += sizeMap.get(i) ?? 0;
```

Comment:
If you wanted this to error, do

```
const size = sizeMap.get(i);
if (size === undefined) { // or == null?
  throw...
}
x += size;
```

As it was previously written, it wouldn't error. X would just become `NaN` and stay `NaN`

Reply:
I'm going to add a `mapGet` method that throws if the value is undefined.